README.md

```diff
@@ -370,7 +371,7 @@ where `synthesize.conf` contains:
 
 --target bins/_Z6popcntm.s # Path to the function to optimize
 
---def_in "{ %rax %rdi }" # The registers that are defined on entry to the target
```

Comment:
per the discussion in #942 including `rax` here incorrectly makes stoke assume it is defined, whereas per the ABI it actually just has garbage